android/src/main/java/io/wazo/callkeep/VoiceConference.java

```diff
@@ -0,0 +1,44 @@
+package io.wazo.callkeep;
```

Comment:
For the record, we started implementing conference, but it was removed from our GA requirements, so this isn't finished yet. This file is the framework for where further implementation will happen.